anubizz-dev/AAD__website
# Added sending contact us messages

Issue: #8 

Description:
Added features to send contact us messages

How to QA:
1. Try sending a contact us message, see that it does not have an error.
2. after sending message @iamariezflores to see if the message was received in the backend.
